src/SimpleAmqpClient/Channel.h

```diff
@@ -523,8 +523,9 @@ class SIMPLEAMQPCLIENT_EXPORT Channel : boost::noncopyable {
    * overload
    * doesn't require the Envelope object to Acknowledge
    * @param delivery_info
+   * @param multiple if true, ack all messages up to this delivery tag, if false ack only this delivery tag
    */
-  void BasicAck(const Envelope::DeliveryInfo &info);
+  void BasicAck(const Envelope::DeliveryInfo &info, bool multiple = false);
```

Comment:
Please create a separate member function overload instead of adding a default parameter.  This will make this less likely to cause an ABI break.

Reply:
Done